### PR TITLE
[bitnami/pgbouncer] Socket+log options and few connections

### PR DESCRIPTION
**Pull request by wapmorgan**
Add passing socket & log options, and passing as much as needs connections to pgbouncer via DATABASE_0_DSN, DATABASE_1_DSN, and so on in pgbouncer `[databases]` list:

```ini
[databases]
foodb = host=host1.example.com port=5432
bardb = host=localhost dbname=bazdb
* = host=foo
```
it will look like (in docker-compose.yml):
```yml
  pgbouncer:
    image: bitnami/pgbouncer
    environment:
      - DATABASE0_DSN=foodb=host=host1.example.com port=5432
      - DATABASE1_DSN=bardb=host=localhost dbname=bazdb
      - DATABASE2_DSN=*=host=foo
```
Dont forget to remove spaces before and after **=** sign.

So you can now pass 2, 5 and even 100 connections.

Signed-off-by: Sergey <wapmorgan@gmail.com>

### Description of the change

Change in entrypoint script.

### Benefits

Add ability to: 
- pass unix socket options
- pass log options
- pass as much as needs connections for pgbouncer.

### Possible drawbacks

none.

### Applicable issues

none.

### Additional information

none.

### CODE REVIEW

File: bitnami/pgbouncer/1/debian-11/rootfs/opt/bitnami/scripts/libpgbouncer.sh [hunk 1]
```diff
@@ -217,9 +217,13 @@ pgbouncer_initialize() {
             database_value+=" connect_query='${PGBOUNCER_CONNECT_QUERY}'"
         fi
         ini-file set --section "databases" --key "$PGBOUNCER_DATABASE" --value "$database_value" "$PGBOUNCER_CONF_FILE"
+        i=0; while true; VAR_NAME="DATABASE${i}_DSN"; do if [ -z "${!VAR_NAME+x}" ]; then break; else dsn=${!VAR_NAME}; echo \$dsn; ini-file set --section databases --key "$(echo $dsn | cut -d = -f 1)" --value "$(echo $dsn | cut -d = -f 2-)" "$PGBOUNCER_CONF_FILE"; i=$(( $i + 1 )); fi; done;
```
Comment (corico44) [inline bitnami/pgbouncer/1/debian-11/rootfs/opt/bitnami/scripts/libpgbouncer.sh:220]: Could you separate the code into different lines to make it clearer, please?

Comment (wapmorgan) [reply bitnami/pgbouncer/1/debian-11/rootfs/opt/bitnami/scripts/libpgbouncer.sh]: Done.
But I haven't tested bash syntax correctness

Comment (corico44) [reply bitnami/pgbouncer/1/debian-11/rootfs/opt/bitnami/scripts/libpgbouncer.sh]: I will check it. Thank you.

**Comment by corico44** [comment]
Hello @wapmorgan,

I have reviewed and tested the logic you propose but it fails as the environment variables you want to use don't exist. You should add them in **_bitnami/pgbouncer/1/debian-11/rootfs/opt/bitnami/scripts/pgbouncer-env.sh_**file.

**Comment by wapmorgan** [comment]
@corico44 that variables should be passed by developer via docker / docker compose params. Pgbouncer should not set them by itself

**Comment by wapmorgan** [comment]
Its additional functionality, not the main one, when you need to pool connections to few servers.

**Comment by wapmorgan** [comment]
@corico44 understood you.
Added envs to file that should be in file.

Also change DSN vars name to PGBOUNCER_DSN_0, 1, ...

**Comment by corico44** [comment]
Thank you @wapmorgan, but the PGBOUNCER_SOCKET_DIR variable is still not declared in the pgbouncer-env.sh file.

**Comment by corico44** [comment]
You are declaring some variables which are already in the original code, like **PGBOUNCER_SERVER_IDLE_TIMEOUT** and **PGBOUNCER_QUERY_WAIT_TIMEOUT**. Could you remove the variables you added that already existed, please?

**Comment by wapmorgan** [comment]
Updated initiation

**Comment by mZarczynski** [comment]
hi, may I count on some progress here? I need  PGBOUNCER_LOG_CONNECTIONS and PGBOUNCER_LOG_DISCONNECTIONS envvars working to avoid creating ini file from scratch and I'm not sure if should I create another PR just to handle these two.

**Comment by corico44** [comment]
@mZarczynski Since @wapmorgan isn't active to be able to apply the changes that were suggested, I proceed to make them myself so that this PR can be accepted. You will be notified in this PR when the changes are merged.

**Comment by wapmorgan** [comment]
Im active, but have not noticed invalid variable definition.
Thank you

**Review by corico44 — APPROVED** [review]
LGTM

**Comment by corico44** [comment]
@mZarczynski changes applied! @wapmorgan thank you for the contribution.

**Pull request merged**